pkg/codegen/internal/test/helpers.go

```diff
@@ -345,6 +344,7 @@ func RunCommand(t *testing.T, name string, cwd string, exec string, args ...stri
 		if len(stderr) > 0 {
 			t.Logf("stderr: %s", stderr)
 		}
+		require.NoError(t, err)
```

Comment:
We can replace the `require.NoError(t, err)` and the existing `if` with a call to `assert.NoError` and a check on its return value:

```suggestion
		if !assert.NoError(t, err) {
```

`assert` does not fail a test promptly; `require` does.